plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/BaseOracleCastPushdownTest.java

```diff
@@ -273,16 +273,6 @@ public void testCastPushdownDisabled()
                 .isNotFullyPushedDown(ProjectNode.class);
     }
 
-    @Test
```

Comment:
We should merge `BaseOracleCastPushdownTest` into `TestOracleCastPushdown` instead of moving this test. Having an abstract layer for single test class doesn't make sense.